### ISSUE TITLE
Cannot find symbol javax.annotation.CheckForNull when trying to build in IntelliJ

### ISSUE THREAD

**Issue by tiwoc** [dropbox/djinni#198]
I get a lot of errors when loading Djinni in IntelliJ and trying to build. Here's an example:

```
/path/to/djinni/test-suite/generated-src/java/com/dropbox/djinni/test/AssortedPrimitives.java
Error:(6, 24) java: cannot find symbol
  symbol:   class CheckForNull
  location: package javax.annotation
```

I'm using IntelliJ IDEA 15 community edition with OpenJDK 1.8.0_71.

I already figured out that the nullability annotations are not shipped with the JDK but have been introduced with JSR 305. I have tried to add an implementation of the JSR by adding `	"com.google.code.findbugs" % "jsr305" % "3.0.0"` to the libraryDependencies in src/build.sbt, but this doesn't seem to do anything.

**Comment by artwyman** [comment]
The required jars for building the Djinni-generated code and utests can be found in djinni/deps/java/

I've used IntelliJ to develop on Djinni, but mostly to build and run the Scala code.  I do have IntelliJ set up so it doesn't complain about those particular failures, though I guess the setting I used to do so must have turned out to be one of the local/gitignored files, not a file in the repo.

If you're an IntelliJ user and want to improve the state of the IntelliJ project files in Djinni I'd welcome a pull request.  I think I'm the only one I can confirm to be using them right now, and I'm not an IntelliJ expert by any stretch.
